extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -87,13 +87,29 @@ export class AmpConsent extends AMP.BaseElement {
 
     /** @private {!Object<string, function()>} */
     this.dialogResolver_ = map();
+
+    /** @private {!Object<string, boolean>} */
+    this.consentUIPendingMap_ = map();
```

Comment:
Should this be part of the UI manager?

Reply:
Hmmm UI manager is designed in a way that queues notification UI in a row. I think this is a good point. It is ideal to make UI manager smart enough to only add notification UI to queue if not currently queued up. 
The thing is only UI from `<amp-consent>` can be re added to the UI manager. I guess it's OK for now. I don't have a good design to represent a notification UI. any ideas?